CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+## Overview
+
+This document explains the processes and practices recommended for contributing enhancements to
+this operator.
+
+- Generally, before developing enhancements to this charm, you should consider [opening an issue
+  ](https://github.com/canonical/ory-kratos/issues) explaining your use case.
+- If you would like to chat with us about your use-cases or proposed implementation, you can reach
+  us at [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
```

Comment:
I wonder if it's better to address to charm-dev or our team, if we'll have a public channel? We may want to specify who to reach out to, depending if it concerns kratos or charm development.

Reply:
That's the public mattermost. It seems like it only has that one channel. I will reword it a little bit